AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -661,7 +666,7 @@ def _lease_task(self, index, new_eta):
     parameters = {'app': self.app, 'queue': self.name, 'id': task_id}
     result = self.db_access.session.execute(select_task, parameters)[0]
     task_info = {
-      'queueName': self.name,
+      'queueName': LONG_QUEUE_FORM.format(app=self.app, queue=self.name),
```

Comment:
I think the substitution should happen inside `rest_api.py` since this class is shared between the rest and protobuf servers.